src/client/components/app/LineDetailsDisplay/PositionsTable.tsx

```diff
@@ -283,9 +293,9 @@ export const PositionsTable = (props: PositionsProps) => {
                   actions={
                     event['status'] === 'PROPOSED' && userRoleMetadata.role === BORROWER_POSITION_ROLE
                       ? [ApproveMutualConsent]
-                      : userRoleMetadata.role === LENDER_POSITION_ROLE && event['status'] === 'OPENED'
+                      : isWithdrawable(event['deposit'], event['principal'], event['lender'])
```

Comment:
I think we need to find a way to refactor this so it doesn't use the nested ternary operators. @kibagateaux thoughts?

Reply:
A get function sounds the best, will work on that